### PR TITLE
Fix problem with enl extensions

### PR DESCRIPTION
**Pull request by GliderGeek**
The current IGC reader does not correctly parse the record extensions. This branch exposes this bug and fixes it.

### CODE REVIEW

File: tests/igc/test_reader.py [hunk 1]
```diff
@@ -32,6 +32,28 @@ def test_decode_B_record():
...
+    # split up per 10 to enable easy counting
+    b_record = 'B093232520' + '2767N00554' + '786EA00128' '0019600600' '1145771529' + '3177005930' + '2770090'
+
+    expected_enl = int(b_record[38:41])  # because of pythons 0 indexing
+
+    decoded_b_record = LowLevelReader.decode_B_record(b_record)
+    processed_b_record = LowLevelReader.process_B_record(decoded_b_record, fix_record_extensions)
+
+    assert 'ENL' in processed_b_record
+    assert expected_enl == processed_b_record['ENL']
```
Comment (Turbo87) [inline tests/igc/test_reader.py]: we should be asserting against a constant here to make sure we don't screw up our own test logic here. if I read correct `00128` and `00196` are the two altitudes and after that are the extensions. the first extension is `FXA` with `006` and `ENL` is second with `001` which is `1` as an integer.

in other words, I think this should just be:
```python
assert processed_b_record['ENL'] == 1
```

**Comment by Turbo87** [comment]
oh, looks like we have a small merge conflict now 😞

**Comment by GliderGeek** [comment]
Any action from me needed?

**Comment by Turbo87** [comment]
nope, just needed to wait until CI passed and then forgot to check again 🙈

**Comment by GliderGeek** [comment]
I know in gitlab there’s an automated merge when pipeline succeeds functionality. Would be nice if github has this as well

**Comment by Turbo87** [comment]
totally, yeah... 😞

**Pull request merged**